neuralmonkey/decoders/sequence_labeler.py

```diff
@@ -94,7 +94,8 @@ def logits(self) -> tf.Tensor:
             embedded_inputs, dweights_4d, [1, 1, 1, 1], "SAME")
         dmultiplication_3d = tf.squeeze(dmultiplication, squeeze_dims=[2])
 
-        logits = multiplication_3d + dmultiplication_3d + biases_3d
+        logits = multiplication_3d + dmultiplication_3d
+        logits = logits + biases_3d
```

Comment:
+=

Reply:
to se nevejde na jednu řádku?

Reply:
Jo, je mozne, ze mi to tam zustalo po debugovani.